### PR TITLE
added an example for merging resource types

### PR DESCRIPTION
**Pull request by sichvoge**
closing #322

### CODE REVIEW

File: versions/raml-10/raml-10.md [hunk 1]
```diff
@@ -2204,6 +2204,41 @@ Finally, the resource can have its own traits, and it can be applied a chain of
...
+  type: collection
+  get:
+    description: override the description
+    responses:
+      200:
+        body:
+          application/json:
+```
+
+The only overlay between the `collection` resource type and the resource declaration is `description` which is defined in both. In this example, the final version will have the description that has been explicitly defined in the resource.
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: overlap, not overlay

File: versions/raml-10/raml-10.md [hunk 2]
```diff
@@ -2204,6 +2204,41 @@ Finally, the resource can have its own traits, and it can be applied a chain of
...
+
+Every explicit node will win over the ones that are declared in a resource type or trait. The rest get's simple merged. The final merged result must be:
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: "The rest get's simple merged" -- you mean "the rest is simply merged"? In any case there is no apostrophe (').

Comment (usarid) [inline versions/raml-10/raml-10.md]: I hope we have more formal rules somewhere than statements like "winning over" :-)

**Comment by usarid** [comment]
LGTM after taking note of a couple of comments.

File: versions/raml-10/raml-10.md (hunk 2 above)
Comment (sichvoge) [inline versions/raml-10/raml-10.md]: They are in the merging algorithm that was in there since RC1.

**Pull request merged**